README.md

```diff
@@ -19,3 +19,12 @@ https://welcome.realdevsquad.com/discord
 # Code of conduct at RDS
 
 https://welcome.realdevsquad.com/code-of-conduct.html
+
+# Running the project
+
+- Run `npm install` after opening the repository in your editor. This will install all the necessary packages for the project which are mentioned in the `package.json` file of the project.
+
+# Scripts used in project
+
+- `npm run check` checks the formatting of all the files in the project. This script runs in a pre-commit hook. It will warn about the improper formatting present in the file.
+- `npm run fix` can be run for fixing the formatting before commiting your changes.
```

Comment:
We can also mention that if the devs get an error during commiting, they can use this script to get rid of the erros.

Reply:
Okay!!